pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -299,6 +299,31 @@ type ClusterStatus struct {
 	CephVersion *ClusterVersion `json:"version,omitempty"`
 }
 
+// CephDaemonsVersions is a structure that can be used to parsed the output of the 'ceph versions' command
+type CephDaemonsVersions struct {
+	// Mon show Mon Ceph version
+	// +optional
+	Mon map[string]int `json:"monVersion,omitempty"`
+	// Mgr show Mgr Ceph version
+	// +optional
+	Mgr map[string]int `json:"mgrVersion,omitempty"`
```

Comment:
the json naming conventions could be a bit more consistent. Some daemon names are prefixed with `versions` (`osdVersion` , `mgrVersion` etc)  and some are not ( `mds`, `rgw` etc) .  I would suggest not to use the `version` prefix at all.

Reply:
removed.